homestyle.css

```diff
@@ -0,0 +1,908 @@
+body {
+    margin:0;
+    padding:0; 
+    text-decoration: none; 
+    max-width:100%;
+    height:100%;
+    overflow-x: hidden;
+    padding-bottom:-200;
+    
+}
```

Comment:
There is a lot going on in here that is causing issues for the rest of your code. You then try to fix the issues established here by doing more CSS on the sections that weren't lining up correctly. That just puts a band-aid on the problem when we want to fix what the issues was at the start.